systemservices/marketplace/src/contracts/utils.ts

```diff
@@ -1,12 +1,13 @@
-import Web3 from "web3"
-import BigNumber from "bignumber.js";
-import Contract from "web3/eth/contract";
-import { TaskInputs } from "mesg-js/lib/service";
+import Web3 from 'web3'
+import BigNumber from 'bignumber.js';
+import Contract from 'web3/eth/contract';
+import { TaskInputs } from 'mesg-js/lib/service';
+const base58 = require('base-x')('123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz')
```

Comment:
Are we sure about that ?
It seems some character are missing.
`0` `I` `O` `l`
Seems obvious that is to avoid error with `0O` and `Il` but any reason to totally remove them ?

Reply:
https://en.wikipedia.org/wiki/Base58